src/arithmetic/placeholder_funcs/placeholder_funcs.c

```diff
@@ -8,7 +8,7 @@
 #include "../func_desc.h"
 #include "../../util/arr.h"
 
-SIValue AR_PATH_FILTER(SIValue *argv, int argc) {
+SIValue AR_PATH_FILTER(SIValue *argv, int argc, void *private_data) {
```

Comment:
is this function still necessary ?

Reply:
yes
in src/ast/ast_build_filter_tree.c _convertPatternPath